src/index.js

```diff
@@ -89,6 +91,15 @@ export default class TronWeb extends EventEmitter {
 
         this.fullnodeVersion = DEFAULT_VERSION;
         this.injectPromise = injectpromise(this);
+
+        this.tronGrid = new TronGrid(this);
+        this.useTronGridAPI = options.useTronGridAPI || false;
```

Comment:
is this.useTronGridAPI variable really necessary? We could just return false as default value if options.useTronGridAPI is not a boolean.

Reply:
If we don't set the default in `useTrongrid` method we should return false if it is unset.